app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List
+
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: List[Player]) -> float:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elfs: List[Elf]) -> None:
+    [elf.play_elf_song() for elf in elfs]
```

Comment:
you don't need comprehension here, cuz you don't even use list which is created. So it is just a memory waste